app/javascript/miq-redux/store.ts

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose, Store} from 'redux';
+import {rootReducer} from './reducer';
+
+import {AppState} from '../packs/miq-redux';
+
+const composeEnhancers = window['.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
```

Comment:
Good one. Did not used the browser's febugger, yet. But I'd definetely wonder why it's not working.